courses/templates/course_detail.html

```diff
@@ -101,6 +101,29 @@ <h2>{{ course.outcomes.sub_heading | safe }}</h2>
              </div>
         </div>
     {% endif %}
+    {% if course.who_should_enroll %}
+      <div class="who-should-enroll-block">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-6 {% if course.who_should_enroll.switch_layout %}order-2{% endif %}">
+              <h1 class="text-uppercase">Who Should Enroll</h1>
```

Comment:
currently we have a class `uppercase` in `static/scss/detail/common.scss`, It would be great if you please remove that and use this `text-uppercase` from bootstrap.

Reply:
Removed from common, `_faqs.html` updated that was using it